src/test/java/org/mule/test/http/functional/requester/AbstractHttpRedirectTimeoutTestCase.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.test.http.functional.requester;
+
+import static java.lang.String.format;
+import static java.lang.String.valueOf;
+import static org.mule.test.http.AllureConstants.HttpFeature.HttpStory.TIMEOUT;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import io.qameta.allure.Feature;
+import org.eclipse.jetty.server.Request;
+import org.junit.Rule;
+
+import org.mule.tck.junit4.rule.SystemProperty;
+
+@Feature(TIMEOUT)
```

Comment:
Story, Feature is HTTP (check whether this is present in a superclass and add it if not, you can check other tests cases for reference)